src/Concerns/DealsWithTenantSymlinks.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Concerns;
+
+use Illuminate\Support\Collection;
+use Stancl\Tenancy\Database\Models\Tenant;
+
+trait DealsWithTenantSymlinks
+{
+    /**
+     * Get all possible tenant symlinks, existing or not (array of ['public path' => 'storage path']).
+     * This is used for creating all possible tenant symlinks and removing all existing tenant symlinks.
+     *
+     * @return array
+     */
+    protected static function possibleTenantSymlinks(Tenant $tenant): Collection
+    {
+        $diskUrls = config('tenancy.filesystem.url_override');
+        $disks = config('tenancy.filesystem.root_override');
+        $suffixBase = config('tenancy.filesystem.suffix_base');
+        $symlinks = collect();
+        $tenantKey = $tenant->getTenantKey();
+
+        foreach ($diskUrls as $disk => $publicPath) {
+            $storagePath = str_replace('%storage_path%', $suffixBase . $tenantKey, $disks[$disk]);
+            $publicPath = str_replace('%tenant_id%', $tenantKey, $publicPath);
+
+            $symlinks->push([public_path($publicPath) => storage_path($storagePath)]);
+        }
+
+        return $symlinks->mapWithKeys(fn ($item) => $item);
+    }
+
+    /**
+     * Determine if the provided path is an existing symlink.
+     */
```

Comment:
```suggestion
    /** Determine if the provided path is an existing symlink. */
```

FYI we use this style for docblocks that don't have any params or return types.

Reply:
Keeping this unresolved so that you see it @lukinovec. You can resolve this, I already committed the change.